x-pack/plugins/spaces/public/views/space_selector/_space_selector.scss

```diff
@@ -0,0 +1,70 @@
+// @import "~ui/styles/variables";
+
+// #spaceSelectorRootWrap, #spaceSelectorRoot {
+//   background-color: $euiColorLightestShade;
+// }
+
+// #spaceSelectorRootWrap {
+//   flex-grow: 1;
+// }
+@import '@elastic/eui/src/components/form/variables';
+
+.spcSpaceSelector {
+  padding: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: $euiZLevel9 + 1000;
```

Comment:
Assume this is because it loads as an overlay?

Reply:
Not sure I just copied straight from https://github.com/elastic/kibana/blob/e08373d93c98e4a7917e4a999616774361b3fbc7/src/core_plugins/kibana/public/home/components/_welcome.scss#L8